chainer/optimizer_hooks/gradient_lars.py

```diff
@@ -0,0 +1,83 @@
+from chainer import cuda
+
+
+class GradientLARS(object):
+
+    """Optimizer/UpdateRule hook function for layer wise adaptive rate scaling.
+
+    See: `Large Batch Training of Convolutional Networks \
+          <https://arxiv.org/abs/1708.03888>`_
+       : `Convergence Analysis of Gradient Descent Algorithms \
+          with Proportional Updates \
+          <https://arxiv.org/abs/1801.03137>`_
+
+    This hook function scales all gradient arrays to fit to the weight norm.
+
+    In <https://arxiv.org/abs/1708.03888>
+        \\Delta w_{t+1} = m * \\Delta w_t
+                        + \\gamma * \\lambda * (\\nabla L_t(w) + \\beta w_t)
+        where
```

Comment:
`lambda` is a time-dependent variable as it is a function of the current weight `w_t`. But so strictly speaking, it may be better to write as `lambda_t`. But current document is still OK because the original paper (https://arxiv.org/abs/1708.03888) also write this way.